Button/button.css

```diff
@@ -0,0 +1,180 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root{
+    --gray:#3e4152;
+    --text-color:#007FFF;
+    --gray-bg:#e5e5e5;
+    --gray-hover:#adb5bd;
+    --blue-bg:#023e8a;
+    --bg-code:#FAFAFA;
+}
+
+body{
+    font-family: 'Poppins', sans-serif;
+    margin: 2rem;
+    color:var(--gray);
+}
+h1{
+    font-size:3rem;
+}
+
+h1, h2, p{
+    margin: 0.625rem 0;
+    line-height: 1.2;
+}
+
+h2{
+    font-size: 2rem;
+}
+p{
+    font-size: 1.2rem;
+    padding: 0.8rem;
+}
+.btn-box{
+    border:1px solid var(--gray);
+    margin: auto;
+    width:80%;
+    padding: 0.2rem;
+    border-radius: 6px;
+    margin-bottom: 1.5rem;
+}
+.fas{
+    font-size: 1rem;
+    padding: 0.4rem;
+}
+
+button{
+    font-size: 1rem;
+    border: none;
+    outline: none;
+    background: white;
+    cursor: pointer;
+}
+
+.btn-variant{
+    margin: 1.5rem 0.8rem;
+    border-radius: 6px;
+}
+.btn-solid-primary, .btn-floating{
+    color:white;
+    background: var(--text-color);
+    border: white;
```

Comment:
I guess 'border':'white' won't make any changes here. 'border-style' should be required.